### PR TITLE
Add package : MHHDoS 

### PR DESCRIPTION
**Pull request by PardesiCat**
Requesting you to add [MHDDoS](https://github.com/MatrixTM/MHDDoS) a open source ddos attack tool in blackarch
i already wrote the PKGBUILD    
 + I used official pkgbuild template
+ its python3 based tool
+ template used [PKGBUILD-python-standalone](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-standalone)

you can check my pkgbuild code here separately for easy inspect 

+ [My pkgbuild ](https://github.com/PardesiCat/blackarch-ddosattck-pkgbuild)
+ [My pkgbuild on my own git server](https://git.pardesicat.xyz/pardesicat/blackarch-ddosattck-pkgbuild)

### CODE REVIEW

**Comment by noraj** [comment]
Generically speaking we don't package DDoS tools, I think the only exception in our repository is Ddosify. Because those tools are very rarely used for ethical or legit purpose and more generally used only by script kiddies for illegal activities.

I'll let the others judge if we accept this tool or not and just technically review your PR in the meanwhile. If we refuse DDoS tools I'll update #2842 to prevent people PRing this kind of tools in the future.

File: packages/MHDDoS/MHDDoS.install [hunk 1]
```diff
@@ -0,0 +1,13 @@
+post_install() {
```
Comment (noraj) [inline packages/MHDDoS/MHDDoS.install:1]: there can't be uppercase in the file path

File: packages/MHDDoS/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+
+pkgname=MHDDoS
+pkgver=2.4.1
+pkgrel=1
+pkgdesc='Best DDoS Attack Script Python3, (Cyber / DDos) Attack With 56 Methods.'
```
Comment (noraj) [inline packages/MHDDoS/PKGBUILD:6]: description shouldn't be promotional eg. using _best_ but rather neutral

File: packages/MHDDoS/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+arch=('any')
+groups=('blackarch' 'blackarch-dos')
+url='https://github.com/MatrixTM/MHDDoS'
+license=('MIT')
+depends=('python3')
```
Comment (noraj) [inline packages/MHDDoS/PKGBUILD:11]: this dependency doesn't exist

```suggestion
depends=('python')
```

File: packages/MHDDoS/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+
+pkgname=MHDDoS
```
Comment (noraj) [inline packages/MHDDoS/PKGBUILD:3]: package name must be lower cased

File: packages/MHDDoS/MHDDoS.install [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  set -e
+  cd /usr/share/MHDDoS
+  python -m venv venv
+  source venv/bin/activate &&
+    pip install --isolated --root="/usr/share/MHDDoS" --prefix='venv' \
```
Comment (noraj) [inline packages/MHDDoS/MHDDoS.install:6]: looking at https://github.com/MatrixTM/MHDDoS/blob/main/requirements.txt this package doesn't seem to require a virtual environment and a jointscript, all dependencies are in AL or BA

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/MHDDoS/PKGBUILD (hunk 2 above)
Comment (PardesiCat) [reply packages/MHDDoS/PKGBUILD:6]: i copied exactly fro the discription  becoz tool owner is not mine  but i will keep it mind

File: packages/MHDDoS/PKGBUILD (hunk 3 above)
Comment (PardesiCat) [reply packages/MHDDoS/PKGBUILD:11]: aa sorry i  am new to pkgbuild   i saw python2 ended support so i put 3  .. i will take note

File: packages/MHDDoS/PKGBUILD (hunk 4 above)
Comment (PardesiCat) [reply packages/MHDDoS/PKGBUILD:3]: oke got you.......

File: packages/MHDDoS/MHDDoS.install (hunk 5 above)
Comment (PardesiCat) [reply packages/MHDDoS/MHDDoS.install:6]: ayy got it  

but we need this git+https://github.com/MHProDev/PyRoxy.git
i think its not avalable in BA or AR

Comment (noraj) [reply packages/MHDDoS/MHDDoS.install:6]: You're right, tihs one is not, in this case it need to be packaged with this template https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools, it should be pretty straightforward.

**Comment by PardesiCat** [comment]
> I'll let the others judge if we accept this tool or not and just technically review your PR in the meanwhile. If we refuse DDoS tools I'll update https://github.com/BlackArch/blackarch/issues/2842 to prevent people PRing this kind of tools in the future.

any update on this ?   or should i continue to the work as per review  changes and again do the PR?

**Comment by noraj** [comment]
You can already do the changes previously discussed on a new commit to update this PR.

**Pull request closed without merging**